src/commands/faq.py

```diff
@@ -0,0 +1,23 @@
+from discord import app_commands, Interaction
+
+
+class FaqGroup(app_commands.Group):
```

Comment:
@phoenixpereira do you think it should be named `FAQGroup` or `FaqGroup`?

Reply:
Probably better to make it `FAQGroup`, but the actually command names can only be lowercase so it would still be `faq` in line 6.

Reply:
@TalkingF can you rename `FaqGroup` to `FAQGroup` then? Otherwise, everything else is looking good